### PR TITLE
Oc 429 remove activate device request

### PR DESCRIPTION
**Pull request by smvdheijden**

### CODE REVIEW

File: osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/infra/jms/ws/messageprocessors/ActivateDeviceRequestMessageProcessor.java [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-@Component
-public class ActivateDeviceRequestMessageProcessor extends WebServiceRequestMessageProcessor {
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(ActivateDeviceRequestMessageProcessor.class);
-    @Autowired
-    @Qualifier("domainAdminDeviceManagementService")
-    private DeviceManagementService deviceManagementService;
-
-    public ActivateDeviceRequestMessageProcessor() {
-        super(DeviceFunction.ACTIVATE_DEVICE);
```
Comment (bvdzwet) [inline osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/infra/jms/ws/messageprocessors/ActivateDeviceRequestMessageProcessor.java]: Should the DeviceFunction values for `ACTIVATE_DEVICE` and `DEACTIVATE_DEVICE` be removed as well?

File: osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/application/services/DeviceManagementService.java [hunk 2]
```diff
@@ -104,10 +103,9 @@ public void handleUpdateKeyResponse(final String deviceIdentification, final Str
             osgpException = new TechnicalException(ComponentType.UNKNOWN, "Exception occurred while updating key", e);
         }
 
-        ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
+        final ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
                 .withCorrelationUid(correlationUid).withOrganisationIdentification(organisationIdentification)
-                .withDeviceIdentification(deviceIdentification).withResult(result).withOsgpException(exception)
-                .build();
+                .withDeviceIdentification(deviceIdentification).withResult(result).withOsgpException(exception).build();
```
[the remark this answers is not in the document]
Comment (smvdheijden) [reply osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/application/services/DeviceManagementService.java]: You're right, this issue was probably caused by the "too many parameters" refactorings...
I further noticed that message type parameter is not used, shouldn't this also be added to the refactored ResponseMessage code?

File: osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/application/services/DeviceManagementService.java [hunk 3]
```diff
@@ -166,42 +164,10 @@ public void handleRevokeKeyResponse(final String organisationIdentification, fin
...
-        final Device device = this.deviceRepository.findByDeviceIdentification(deviceIdentification);
-
-        device.setDeviceLifecycleStatus(deviceLifecycleStatus);
-
-        this.deviceRepository.save(device);
-
         final ResponseMessage responseMessage = ResponseMessage.newResponseMessageBuilder()
                 .withCorrelationUid(correlationUid).withOrganisationIdentification(organisationIdentification)
-                .withDeviceIdentification(deviceIdentification).withResult(ResponseMessageResultType.OK).build();
+                .withDeviceIdentification(deviceIdentification).withResult(result).withOsgpException(exception).build();
```
[the remark this answers is not in the document]
Comment (smvdheijden) [reply osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/application/services/DeviceManagementService.java]: You're right, this issue was probably caused by the "too many parameters" refactorings...
I further noticed that message type parameter is not used, shouldn't this also be added to the refactored ResponseMessage code?

File: osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/infra/jms/ws/messageprocessors/ActivateDeviceRequestMessageProcessor.java (hunk 1 above)
Comment (smvdheijden) [reply osgp-adapter-domain-admin/src/main/java/com/alliander/osgp/adapter/domain/admin/infra/jms/ws/messageprocessors/ActivateDeviceRequestMessageProcessor.java]: You're right, I've removed them

**Pull request merged**